lndr-backend/src/Lndr/Types.hs

```diff
@@ -224,11 +224,19 @@ data EthereumPrices = EthereumPrices { usd :: Double
                                      , nok :: Double
                                      , sek :: Double
                                      , nzd :: Double
+                                     , idr :: Double
+                                     , myr :: Double
+                                     , sgd :: Double
+                                     , thb :: Double
+                                     , vnd :: Double
+                                     , ils :: Double
+                                     , rub :: Double
+                                     , try :: Double
                                      } deriving (Show, Generic)
 $(deriveToJSON defaultOptions ''EthereumPrices)
 
 instance Default EthereumPrices where
-    def = EthereumPrices 420 45000 460000 2500 400 2600 340 550 300 3000 540 3250 3450 575
+    def = EthereumPrices 640 70018 690713 3939 633 4056 529 847 459 5022 824 5117 5545 906 8891939 2508 850 20240 14580140 2294 40134 2603
```

Comment:
🤔This line was concerning before, and now it's even moreso with its ever-increasing sequence of magic numbers.